### PR TITLE
#53 ADD docs for viz.df_utils.py

### PR DESCRIPTION
**Pull request by jalew188**

### CODE REVIEW

File: nbs/test_viz.ipynb [hunk 1]
```diff
@@ -2,7 +2,7 @@
  "cells": [
```
Comment (mschwoer) [inline nbs/test_viz.ipynb:2]: so there was no valuable wisdom in this notebook that was worth preserving? :-)

**Review by mschwoer — APPROVED** [review]

File: nbs/test_viz.ipynb (hunk 1 above)
Comment (jalew188) [reply nbs/test_viz.ipynb:2]: As I said before, the file is too large with plotly plots

Comment (mschwoer) [reply nbs/test_viz.ipynb:2]: then why not just strip the output? :-)

Comment (jalew188) [reply nbs/test_viz.ipynb:2]: Just click "clear output" button in vscode ...

**Pull request merged**